### ISSUE TITLE
r8942 space between subtitles

### ISSUE THREAD

**Issue by andy144** [Aegisub/Aegisub#78]
Hi, Thanks for the latest update.
I use aegisub for mac and  love to see those volume/zoom sliders working now.
My question would be if anybody could change the behavior of the  new line
witch is sticking to the previous?
I mean to have an option of putting an interval in milliseconds
between a subtitle line and the following line, interval that will automatically appear
after confirming with the Enter or G key.
Now I have to modify each line and it's really Neanderthal. :D

Thanks in advance!

**Comment by andy144** [comment]
Yes, I was talking about that gap.
After writing a line, you press “g” or “Enter” and the next line appears, but at the exact moment when the previous line ended.
I would like an option to declare a gap in milliseconds between lines, after pressing g or Enter.

**Comment by andy144** [comment]
But the gap should appear only if the next line it's  a new one, of course. If the gap is already established it should remain the same ( in case of corrections).

**Comment by andy144** [comment]
The thing is, it's annoying to have lines apeearing just as the previous ones disappeared. Sometimes can't see there is another line that came.  That's why me and many others put a small of gap of 30-80 ms. In these situations. 
Subtitle workshop has that option.

**Comment by Nalapl3** [comment]
These gaps are what makes it look bad though.

**Comment by CoffeeFlux** [comment]
I don't have it handy and am out of town right now, but I wrote a script for this a while back since it's sometimes necessary in professional work. I'll upload it once I get home in a few days.

**Comment by andy144** [comment]
Cool!
Thanks in a million! :)

**Comment by Leinad4Mind** [comment]
Yeah, it's needed in some industry's as television. I normally do the subs
on aegisub and save as ebu stl and then I use spot software and convert the
ebu stl to .pac ( this is the most use format for television in Europe )
and unfortunately the hardware that read the .pac needs at least a 5 frames
gap. Or else shit happens lol

When I do fansubbing I use the automatic timing to join the lines, or else
it will suffer from blink problem what sucks... but for the industry after
that step I add a gap of 5 frames on all joined lines.

In the past I used subtitle edit to create this gap since it's easy on that
software. But I've eventually wrote too an automation for aegisub that does
that.

Dunno if I've that automation here on my GitHub account. I've some scripts,
need to check it when got some time.

Em 22/06/2018 5:58 da tarde, "andy144" <notifications@github.com> escreveu:

Cool!
Thanks in a million! :)

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/Aegisub/Aegisub/issues/78#issuecomment-399510885>, or mute
the thread

**Comment by andy144** [comment]
Don’t forget me with that script. :)

**Comment by andy144** [comment]
@CoffeeFlux  please upload the script if you returned home.
Thanx

**Comment by CoffeeFlux** [comment]
Yep, sorry. I'm back now and I'll clean it up and upload when I get a chance.

**Comment by andy144** [comment]
Oh, I thought you forgot. :)
Thanks very much!

**Comment by kikofez** [comment]
@CoffeeFlux , I'm also interested in your script. Where can I find it? Thanks :)

**Comment by ilko-k** [comment]
@andy144 Why do you bother ? Create your lines as usual and when you reach the end, simply apply a shift to start times only, that's it. Then apply a post-precessor treatment if needed and your subs are ready to be checked.

**Comment by andy144** [comment]
Thanks for the idea. It’s a sort of workaround.
But I still hope for an automation. :)

**Comment by andy144** [comment]
Anybody please upload the script?
It would be alot of time saving for some...

**Comment by MuhammadiJK** [comment]
@CoffeeFlux , So what about your script? Does it even exist? You promised it, but never delivered)

I also would love to have it. That's one of the features that Aegisub should have.
